conda.recipe/meta.yaml

```diff
@@ -15,7 +18,7 @@ requirements:
     - python
     - distribute
     - setuptools
-    - nodejs >=4.1
+    - nodejs >=4.1  [unix]
```

Comment:
there is a nodejs available for win-64 here: https://anaconda.org/javascript/nodejs/files... just to let you know...